apps/builder/app/builder/features/style-panel/shared/css-value-input/unit-select-options.ts

```diff
@@ -29,6 +30,17 @@ const preferedSorting = [
 
 const visibleLengthUnits = ["px", "em", "rem", "dvw", "dvh"] as const;
 
+/*
+The boxShadow property is interdependent on other values of the proeprty,and cannot function in isolation.
```

Comment:
I would just add property to css-value-input to not show/add keywords
like 
showKeywords  = false

Reply:
This will remove the need of this comment and boxShadow check